UKHomeOffice/sgmr-service
# SPCR-439 Remove change links from cancelled reports

### AC / Description of changes
On a cancelled report - on the CYA page, the change links are still visible and if you click on one and change something it returns the voyage plan to a draft state. 

Remove the change links so that a cancelled report cannot be altered.

### To Test
- Create a voyage plan
- Get to the CYA page
- Change links should be visible and take you to correct pages
- Cancel the voyage plan
- View the cancelled voyage plan
- The change links should not be visible and therefore you should not be able to alter the voyage plan


### Notes

I think the setIsLoading(false) on the useEffect was redundant as it was preventing the loading spinner from appearing on the CYA page

---

**Remember to merge to feature branches not `master`.**
